### PR TITLE
[GH-13049] Add unit tests to "UserInviteCmd" to use testify

### PR DESCRIPTION
**Pull request by larkox**
Summary:
Add unit tests to "UserInviteCmd" to use testify

Link:
Fixes https://github.com/mattermost/mattermost-server/issues/13049

### CODE REVIEW

File: commands/user_test.go [hunk 1]
```diff
@@ -52,3 +52,302 @@ func (s *MmctlUnitTestSuite) TestSearchUserCmd() {
...
+		s.client.
+			EXPECT().
+			GetTeam(argTeam, "").
+			Return(&model.Team{Id: argTeam, Name: resultName}, &model.Response{Error: nil}).
+			Times(1)
+
+		s.client.
+			EXPECT().
+			GetTeamByName(argTeam, "").
+			Times(0)
```
Comment (mgdelacroix) [inline commands/user_test.go]: This expectation can be removed

File: commands/user_test.go [hunk 2]
```diff
@@ -52,3 +52,302 @@ func (s *MmctlUnitTestSuite) TestSearchUserCmd() {
...
+
+		s.client.
+			EXPECT().
+			GetTeamByName(argTeam[1], "").
+			Times(0)
+
+		s.client.
+			EXPECT().
+			GetTeamByName(argTeam[2], "").
+			Times(0)
```
Comment (mgdelacroix) [inline commands/user_test.go]: These two expectations are safe to remove

File: commands/user_test.go [hunk 3]
```diff
@@ -52,3 +52,302 @@ func (s *MmctlUnitTestSuite) TestSearchUserCmd() {
...
+func (s *MmctlUnitTestSuite) TestUserInviteCmd() {
+	s.Run("Invite user to an existing team by Id", func() {
+		printer.Clean()
+		argUser := "example@example.com"
+		argTeam := "teamId"
+
+		s.client.
+			EXPECT().
+			GetTeam(argTeam, "").
+			Return(&model.Team{Id: "teamId"}, &model.Response{Error: nil}).
```
Comment (mgdelacroix) [inline commands/user_test.go]: nit: use the already created variable `argTeam` to relate it with the expected team

```suggestion
			Return(&model.Team{Id: argTeam}, &model.Response{Error: nil}).
```

File: commands/user_test.go (hunk 2 above)
Comment (larkox) [reply commands/user_test.go]: Not sure how I forgot these ones on the previous sweep. Thanks for pointing them out!

**Pull request merged**